routes/foodtrucks.go

```diff
@@ -149,3 +167,114 @@ func GetFoodTrucksHandler(w http.ResponseWriter, r *http.Request) {
 	w.Header().Set("Content-Type", "application/json")
 	json.NewEncoder(w).Encode(foodTrucks)
 }
+
+func PutFoodTrucksHandler(w http.ResponseWriter, r *http.Request) {
+
+	// Checks for food truck ID
+	params := mux.Vars(r)
+	foodTruckID, foodTruckIDExists := params["foodTruckID"]
+	if !foodTruckIDExists {
+		w.WriteHeader(http.StatusBadRequest)
+		return
+	}
+
+	// Get user from context
+	user, userLoggedIn := r.Context().Value(middleware.UserKey).(string)
+
+	fmt.Println(user)
```

Comment:
remove debug statements

Reply:
ah yes

Reply:
oh wait, user has to be used. i tried to replace it with an _ but it wasnt working

Reply:
ended up just replacing "user" with "_"